components/transferForm.tsx

```diff
@@ -394,7 +395,7 @@ const initialProps: {
     fields: {
       field: string;
       label: string;
-      kind?: "textarea" | "input-complete";
+      kind?: "textarea" | "input-complete" | "autocomplete";
```

Comment:
remind me, what's input-complete about?

Reply:
`input-complete` is the autocomplete that pavlo developed

Reply:
I think there are two kind of autocomplete here:
- "input-complete" is for address and alias
- "autocomplete" is for FA2

what about like this:
- "input-complete" -> "address-autocomplete" (maybe change this next time when refactor)
- "autocomplete" -> "fa2-autocomplete"

Reply:
as it's useless currently, let's just update it later